README.md

```diff
@@ -1,13 +1,12 @@
 # Express API Project
 
-Replace this readme with your own information about your project. 
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+This week projects is the start getting familiar with creating API endpoint, mostly using GET method with Express package.
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+The RESTful API naming, the usage of params and query are the two main difficulties of the projects. I am not really sure I have done it correctly either.
 
 ## View it live
 
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+https://goodbookscollection.herokuapp.com/books
```

Comment:
I tried to open your app, but Heroku returns information that this app is not created.

Reply:
Hey Karol, the link to Heroku is restored. Please check it out. Thank you: 
https://goodbookscollection.herokuapp.com/books